test/tests/functional/pbs_cgroups_hook.py

```diff
@@ -471,21 +469,14 @@ def load_hook(self, url):
             a = {'enabled': 'True',
                  'freq': '2',
                  'event': events}
-            rv = self.server.create_import_hook(self.hook_name, a, script,
-                                                overwrite=True)
-            self.assertTrue(rv)
+            self.server.create_import_hook(self.hook_name, a, script,
+                                           overwrite=True)
```

Comment:
I am not convinced that we need a new function just for create and importing hook. we will avoid reading the hook file into a new file and rather move line 466-472 in setUp. And again why we need try-catch for everything rather than taking the output into a variable and using assertTrue.

Reply:
We need this in a try block so that the temp file gets removed within the finally section. That will run regardless of whether the load is successful.